CMakeLists.txt

```diff
@@ -919,6 +919,9 @@ if((NOT ${CMAKE_SYSTEM_NAME} STREQUAL "Android")
   AND("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "aarch64"))
   string(APPEND CMAKE_CXX_FLAGS " -DAT_BUILD_ARM_VEC256_WITH_SLEEF")
   add_definitions(-DAT_BUILD_ARM_VEC256_WITH_SLEEF)
+
+  string(APPEND CMAKE_CXX_FLAGS " -DAT_BUILD_ARM_VECSVE_WITH_SLEEF")
+  add_definitions(-DAT_BUILD_ARM_VECSVE_WITH_SLEEF)
```

Comment:
doesn't this flag also need to be added to the Apple config above? AIUI the Apple M4 is ARMv9.2, which includes SVE. https://en.wikipedia.org/wiki/Apple_M4

Reply:
I am not sure if M4 has SVE. LLVM [Commit](https://github.com/llvm/llvm-project/commit/2b33591386e855cf82a4104092e1a1d2888f32b1#diff-dce31e228995ede8526247039ce5a8871a69a4823869a13ad65f5bf2d074a9a6R524)

Reply:
TIL!